projects/js-packages/analytics/package.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "@automattic/jetpack-analytics",
+	"version": "0.1.0-alpha",
+	"description": "Jetpack Analytics Package",
+	"author": "Automattic",
+	"license": "GPL-2.0-or-later",
+	"dependencies": {
+		"lodash": "4.17.21"
+	},
+	"devDependencies": {
+		"debug": "4.3.1",
+		"jetpack-js-test-runner": "workspace:*",
+		"nyc": "15.1.0"
+	},
+	"type": "module",
+	"exports": {
+		".": "./index.jsx"
```

Comment:
Tried to make the linter happy. I'm not so familiar with JS Packages, but it seems to have wanted an export module for "./index.jsx" - @anomiex this look okay?

Reply:
Yeah, that looks good.

The point of it is that if we were to change line 8 to `^4.17.21` so people weren't forced to use that one specific version of lodash, having the "exports" will convince renovate that this is a library so that's ok.

Reply:
Thanks for the explanation!